source/common/http/alternate_protocols_cache_impl.cc

```diff
@@ -10,33 +12,30 @@ AlternateProtocolsCacheImpl::~AlternateProtocolsCacheImpl() = default;
 
 void AlternateProtocolsCacheImpl::setAlternatives(const Origin& origin,
                                                   const std::vector<AlternateProtocol>& protocols) {
+  // TODO(RyanTheOptimist): Figure out the best place to enforce this limit.
+  ASSERT(protocols.size() < 10);
```

Comment:
prefer having this a defined const in the class.  mind doing so along with the TODO?